contents/docs/surveys/creating-surveys.mdx

```diff
@@ -105,21 +107,40 @@ You can display your survey to specific users based on:
 - **User sends events:** Display a survey to users who have sent a specific event during their session.
 
 <ProductScreenshot
-    imageLight={displayConditionsLight} 
+    imageLight={displayConditionsLight}
     imageDark={displayConditionsDark}
-    alt="Display conditions" 
+    alt="Display conditions"
     classes="rounded"
 />
 
 ### Completion conditions
 
 This enables you to stop your survey once you receive a specific number of responses. Note that because it takes time for PostHog to process responses, this is a rough guideline. Your survey might receive slightly more responses than you set.
 
+## Repeating surveys
+
+By default, surveys are shown once to each user until they either dismiss it or complete it. After that, they won't see the survey again.
+
+You can also configure surveys to repeat at specific intervals, which is useful for gathering feedback from users on a regular basis.
+
+<ProductScreenshot
+    imageLight={repeatingSurveyLight}
+    imageDark={repeatingSurveyDark}
+    alt="Repeating surveys"
+    classes="rounded"
+/>
+
+For example, you can configure settings like:
+- **Repeat this survey `3` times, once every `90` days**: The survey will be shown to a user up to 3 times total, with a 90-day gap between each showing.
```

Comment:
can you write down the example here such as you did below:

> if you set a survey to repeat every 90 days and a user completes it on January 1st, they won't see it again until April 1st

eg, 1st jan until 31 march, 1 April until 30 June, 1st july until 30 sept and after that, never anymore since you configured only 3 times, is that correct?